src/elements/Icon.tsx

```diff
@@ -30,7 +30,7 @@ export function Icon({children, ...props }: Icon<HTMLElement>) {
 
     const icon = (
         <span {...HTMLProps} className={className}>
-            <i className={`${props.className}`} aria-hidden="true"></i>
+            {children ? children : (<i className={`${props.className}`} aria-hidden="true"></i>)}
```

Comment:
I'd really like this PR to get merged. What is the state of this?
Is there anyone to approve?

I think the solution is a good one.

Maybe this would be slightier more readable?

```suggestion
            {children || <i className={`${props.className}`} aria-hidden="true" />}
```

It would fix #126